rpc/src/types/errors.rs

```diff
@@ -0,0 +1,23 @@
+use serde::{Deserialize, Serialize};
+use serde_json::{json, Value};
+use std::collections::HashMap;
+
+#[derive(Serialize, Deserialize)]
+pub struct ResponseError {
+  title: String,
+  description: String,
+  r#type: String,
```

Comment:
Whats `r#type`?

Reply:
This was used to escape the type keyword and use it as a field (there might be a better way).

Reply:
afaik this is the only way